ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -35,7 +30,7 @@ type kerberos_principal_string = string with {
         error("Not a valid principal " + principal);
     };
 
-    realm = pc_r[len-1];
+    realm = pc_r[len - 1];
```

Comment:
Also, please don't just change code to please panlint, if you disagree with what it is saying please open an issue.